admin_manual/installation/php_configuration.rst

```diff
@@ -0,0 +1,132 @@
+===========================
+PHP Modules & Configuration
+===========================
+
+PHP Modules
+-----------
+
+This
+section lists all required and optional PHP modules.  Consult the `PHP manual
+<https://php.net/manual/en/extensions.php>`_ for more information on modules.
+You can
+check the presence of a module by typing ``php -m | grep -i <module_name>``.
+If you get a result, the module is present.
+
+Required:
+
+* PHP (see :doc:`./system_requirements` for a list of supported versions)
+* PHP module ctype
+* PHP module curl
+* PHP module dom
+* PHP module fileinfo (included with PHP)
+* PHP module filter (only on Mageia and FreeBSD)
+* PHP module GD
+* PHP module hash (only on FreeBSD)
+* PHP module JSON (included with PHP >= 8.0)
+* PHP module libxml (Linux package libxml2 must be >=2.7.0)
+* PHP module mbstring
+* PHP module openssl (included with PHP >= 8.0)
+* PHP module posix
+* PHP module session
+* PHP module SimpleXML
+* PHP module XMLReader
+* PHP module XMLWriter
+* PHP module zip
+* PHP module zlib
+
+Database connectors (pick the one for your database:)
+
+* PHP module pdo_sqlite (>= 3, usually not recommended for performance reasons)
+* PHP module pdo_mysql (MySQL/MariaDB)
+* PHP module pdo_pgsql (PostgreSQL)
+
+*Recommended* packages:
+
+* PHP module bz2 (recommended, required for extraction of apps)
+* PHP module intl (increases language translation performance and fixes sorting
+  of non-ASCII characters)
+* PHP module sodium (for Argon2 for password hashing. bcrypt is used as fallback, but if passwords were hashed with Argon2 already and the module is missing, your users can't log in.)
+
+Required for specific apps:
+
+* PHP module ldap (for LDAP integration)
+* PHP module smbclient  (SMB/CIFS integration, see
+  :doc:`../configuration_files/external_storage/smb`)
+* PHP module ftp (for FTP storage / external user authentication)
+* PHP module imap (for external user authentication)
+* PHP module bcmath (for passwordless login)
+* PHP module gmp (for passwordless login)
+
+Recommended for specific apps (*optional*):
+
+* PHP module gmp (for SFTP storage)
+* PHP module exif (for image rotation in pictures app)
+
+For enhanced server performance (*optional*) select one of the following
+memcaches:
+
+* PHP module apcu (>= 4.0.6)
+* PHP module memcached
+* PHP module redis (>= 2.2.6, required for Transactional File Locking)
+
+See :doc:`../configuration_server/caching_configuration` to learn how to select
+and configure a memcache.
+
+For preview generation (*optional*):
+
+* PHP module imagick
+* avconv or ffmpeg
+* OpenOffice or LibreOffice
+
+.. note::
+   If the preview generation of PDF files fails with a "not authorized" error message, you must adjust the imagick policy file.
+   See https://cromwell-intl.com/open-source/pdf-not-authorized.html
```

Comment:
Does it link by default, or do you need to add markup?

Reply:
It links by default:
![Screenshot_20231024_164734](https://github.com/nextcloud/documentation/assets/91878298/b135744a-104e-4c78-b2ec-fb619de60001)